.travis.yml

```diff
@@ -1,13 +1,24 @@
+sudo: required
+dist: trusty
+group: edge
+
 language: go
 go:
     - 1.6.x
     - 1.7.x
     - 1.8.x
     - tip
+
 install:
     - go get -v golang.org/x/tools/cmd/cover
+    - sudo add-apt-repository -y ppa:opencpu/opencpu-1.6
```

Comment:
Yes, the server start code for the local session depends on the `opencpu$start` function. I have written a conditional to sniff the version and decide which start function to call.